### ISSUE TITLE
Remove `active` toggle and log last login

### ISSUE THREAD

**Issue by seblj** [td-org-uit-no/tdctl-frontend#99]
It feels unnecessary to have a toggle for `active` user. It should instead figure this out automatically based on last login for example

**Comment by FredrikMorstad** [comment]
can you give an example, dont think I understand the case

**Comment by seblj** [comment]
Oh I thought the active toggle was still there no matter what, but it looks like it's a one time thing, but I think we should remove it completely and just use the email confirmation as `active`. Seems like an unnecessary extra step to go to the profile just to toggle active when we also have email for confirmation

**Comment by FredrikMorstad** [comment]
yes, however, it is there to fulfill the requirement of an active user set by NT-SU and we can only apply for funding based on these active users
